td-agent/templates/usr/lib/tmpfiles.d/td-agent.conf

```diff
@@ -0,0 +1,4 @@
+d /tmp/td-agent 0755 td-agent td-agent - -
```

Comment:
Should we use `@PACKAGE` or `<%= project_name %>` for generating tempfiles configuration?

Reply:
Ah, It should be, I'll fix it.

Reply:
Fixed.